ofrak_core/ofrak/core/pe/model.py

```diff
@@ -65,7 +65,11 @@ class PeOptionalHeaderMagic(Enum):
 @dataclass
 class PeOptionalHeader(ResourceView):
```

Comment:
Since it does not appear that the order matters for these resource views, I wonder if it would be clearer to have all of the default options be defined in `PeOptionalHeader`.

This would make `Pe32OptionalHeader` simply:

```
@dataclass
class Pe32OptionalHeader(PeOptionalHeader):
    base_of_data: int
```

You could even define `Pe64OptionalHeader` as such:
```
@dataclass
class Pe64OptionalHeader(PeOptionalHeadeR):
```

I found it hard to immediately see the difference between 32/64 bit headers with all of the repetition as it is now.

Reply:
Yeah, I was split because the original has it in the proper order, which made finding certain fields and double checking the structure nicer. I'll change it to this, I think I'll split out the windows specific fields too, even though its not necessary ATM.